JSDemos/Demos/DataGrid/GridColumnsHidingPriority/description.md

```diff
@@ -1 +1,5 @@
-The **DataGrid** can hide columns according to&nbsp;the specified priority if&nbsp;screen width is&nbsp;not enough to&nbsp;display all of&nbsp;them. The hiding priority is&nbsp;specified using the **hidingPriority** field of&nbsp;an&nbsp;item in&nbsp;the **columns** array. Switch between the landscape and portrait layouts to&nbsp;see how the **DataGrid** looks on&nbsp;wide and narrow screens.
\ No newline at end of file
+When the DataGrid adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority).
+
+The lower is this option's value, the sooner a column is hidden. For example, **DataGrid** hides the column whose **hidingPriority** equals *"0"* first, then the column with the priority *"1"* and so on.
+
+In this demo, the `CustomerStoreCity` column has hiding priority *"0"* and is hidden in both vertical and horizontal orientation modes. The `CustomerStoreState` and `OrderDate` columns have priority *"1"* and *"2"* correspondingly. **DataGrid** hides these columns after the `CustomerStoreCity` column when the screen orientation is vertical.
```

Comment:
```suggestion
In this demo, the `CustomerStoreCity` column is hidden in both vertical and horizontal orientation modes because its **hidingPriority** is 0. The `CustomerStoreState` and `OrderDate` columns have priorities 1 and 2. The **DataGrid** hides these columns after the `CustomerStoreCity` column, only when the screen orientation is vertical.
```